app/controllers/application_controller.rb

```diff
@@ -23,4 +23,17 @@ def configure_permitted_parameters
       :profile_picture
     ]
   end
+
+  def after_sign_in_path_for(user)
+     user_path(current_user)
+  end
+
+  def after_sign_up_path_for(user)
+    user_path(current_user)
+  end
+
+  def after_sign_out_path_for(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `after_sign_out_path_for(*)` if you want the method to accept any arguments but don't care about them.